kentikapi/internal/resources/base.go

```diff
@@ -126,3 +134,17 @@ func (b BaseAPI) DeleteAndValidate(ctx context.Context, url string, output inter
 
 	return nil
 }
+
+func LogPayload(l bool, msg, url string, payload interface{}) {
```

Comment:
```suggestion
func (b BaseAPI) logPayload(msg, url string, payload interface{}) {
```

This is local helper function, so it should be un-exported. Also, make it a method to avoid passing a flag every time.

Reply:
Done